contrib/raftexample/raft.go

```diff
@@ -287,9 +287,6 @@ func (rc *raftNode) startRaft() {
 		rc.node = raft.RestartNode(c)
 	} else {
 		startPeers := rpeers
```

Comment:
Thanks for the PR. I agree that we shouldn't call StartNode without peers but in that case you can pass the `rpeers` in your changes without creating startPeers variable now. I haven't played with the test examples here but per the original logic here if rc.join is set to true we should call RestartNode instead of StartNode i.e. may be changing LOC 286 to `if oldwal || rc.join { ...` makes more sense? Also, please modify your `commit title` in this format `<package>{", "<package>}: <description>'` to pass the build. Thanks!